### ISSUE TITLE
Controller (CECH-ZCM1E) not pairing.

### ISSUE THREAD

**Issue by WC372** [Timocop/PSMoveServiceEx#20]
I'm trying to pair my PS Move with PSMoveServiceEx, but I'm getting an error.

The error is:
[2022-11-21 19:28:24.784]: ServerRequestHandler - Failed to start controller(0) stream: Not on stream-able connection.

Does this mean I need to get the Bluetooth adapter or controller on a faster USB connection? I'm using USB 2 and the Asus BT400 bluetooth adapter by the way. I have used a "Hama" branded bluetooth adapter before, but that didn't even want to recognize the PS Move controller.

**Comment by Timocop** [comment]
- Post full log.
 - Show serial number of the controller.
 - Does your computer have build-in bluetooth? If so, disable in device manager.
 - Make sure the Adapter is visible to the controller. (Never attach bluetooth adapters at the back of the computer).

**Comment by WC372** [comment]
[PSMoveService.log](https://github.com/Timocop/PSMoveServiceEx/files/10059611/PSMoveService.log)
 
In PSMoveServiceEx it shows that the device serial is 00:07:04:42:e1:2f If you are looking for the code that's on the back of the controller under the barcode then that is 1131041805375

My computer doesn't have built-in bluetooth and the only things plugged into the front are the bluetooth adapter and the mini-USB cable for the controller.

**Comment by Timocop** [comment]
```PSMoveController::getBTAddress - HID ERROR: The requested resource is in use.```
Seems like there is an issue getting the Bluetooth adapter address. Make sure you dont have any 3th party driver installed for your BT adapter. Try deinstalling it using the device manager. And restart your PC and try again.

**Comment by WC372** [comment]
[PSMoveService2.log](https://github.com/Timocop/PSMoveServiceEx/files/10059918/PSMoveService2.log)
I just tried this on another PC. Still doesn't want to pair, but, this time there's a different log. On my main PC I don't have a third party driver installed but I'll give what you said a try tomorrow since it's getting night here in Europe.

**Comment by Timocop** [comment]
Did you unplug the controller while pairing?
Try unplugging the controller while pairing. Or if that doesn't help, keep the controller plugged in while pairing.
I will have a look at this tomorrow.

**Comment by WC372** [comment]
Okay, I uninstalled the bluetooth adapter drivers, restarted, and then instead of using the drivers that came with the bluetooth adapter (the one on the disc) I used the ones that were on the Asus website and the controller now works. The previous drivers that were on disc were from 2012 but the ones that I installed from the Asus website were from 2021. Anyway, the controller works perfectly now, and I will close this issue. Thank you for the help :)
